src/main/kotlin/uk/gov/justice/digital/hmpps/prison/visitbooker/registry/model/entity/Visitor.kt

```diff
@@ -14,18 +14,18 @@ import java.time.LocalDateTime
 
 @Entity
 @Table(name = "booker_prisoner_visitor")
-class BookerPrisonerVisitor(
+class Visitor(
   @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY)
   @Column(name = "ID")
   val id: Long = 0,
 
   @Column(name = "booker_prisoner_id", nullable = false)
-  val bookerPrisonerId: Long,
+  val prisonerId: Long,
 
   @ManyToOne
   @JoinColumn(name = "booker_prisoner_id", updatable = false, insertable = false)
-  val bookerPrisoner: BookerPrisoner,
+  val prisoner: Prisoner,
```

Comment:
Is the ManyToOne constraint correct?

Can a visitor have multiple prisoners past MVP?